apps/mailrelay/mailrelay/dev/dev.yaml

```diff
@@ -26,8 +26,8 @@ spec:
         objectName: "dev-mailrelay-platform-hmcts-net"
 
     volumeClaim:
-      storageClass: "managed-premium"
-      storageSize: 500Gi
+      storageClass: "standard_ssd"
```

Comment:
this should be `managed`. If you run `kubectl get sc` you can see the options that are there

Reply:
yeah - did saw that but the ticket said this so thougt we need standard ssd?
`We need to update PVCs (total: 4) for Mailrelay to use 64Gb Standard SSD`

Reply:
updated now